far/string_utils.hpp

```diff
@@ -50,10 +50,13 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 constexpr bool IsEol(wchar_t x) noexcept { return x == L'\r' || x == L'\n'; }
 
 [[nodiscard]]
-inline bool IsBlankOrEos(wchar_t x) noexcept { return std::iswblank(x) || !x; }
+inline bool IsBlankOrEos(wchar_t x) { return !x || std::iswblank(x); }
```

Comment:
No noexcept?

Reply:
Yes. `iswblank` / `iswspace` may call `GetStringTypeW`, so who knows whether it can throw. And if it does throw, the process will be terminated with prejudice, which is likely not good.

Reply:
C functions, as well as Win32 APIs, don't throw C++ exceptions by definition.

Reply:
I know; still somehow get caught up in it. :(